src/libraries/Microsoft.PowerFx.Core/Public/Config/ReadOnlySymbolTable.cs

```diff
@@ -290,6 +290,8 @@ public SymbolTable GetMutableCopyOfFunctions()
 
         private protected readonly TexlFunctionSet _functions = new TexlFunctionSet();
 
+        internal virtual Dictionary<TexlFunction, IFunctionImplementation> FunctionImplementations { get; } = new ();

```

Comment:
>IFunctionImplementation [](http://example.com/codeflow?start=50&length=23)

A symbol table should be pure compile-time and not have runtime implementation on it.  So this mapping should probably be elsewhere.